tasks/section5.yml

```diff
@@ -271,15 +393,19 @@
       - rule_5.2.6
 
 - name: "SCORED | 5.2.6 | PATCH | Ensure SSH IgnoreRhosts is enabled"
-  command: /bin/true
+  lineinfile:
+      state: present
+      dest: /etc/ssh/sshd_config
+      regexp: '^IgnoreRhosts'
+      line: 'IgnoreRhosts yes'
   tags:
       - level1
       - level2
       - patch
       - rule_5.2.6
 
 - name: "SCORED | 5.2.7 | AUDIT | Ensure SSH HostbasedAuthentication is disabled"
-  command: /bin/true
+  command: grep "^HostbasedAuthentication no" /etc/ssh/sshd_config
   register: result
```

Comment:
Audit tasks should register verbose variable name that ends with `_audit`. As you are adding AUDIT checks please change `register: result` to `register: `*`descriptive_check_name_audit`*